.gitlab-ci.yml

```diff
@@ -321,16 +321,22 @@ integration:docker:
     - integration:builds
     - job: integration:deployment
       optional: true
-  image: docker:20.10.11
   services:
-    - docker:20.10.11-dind
+    - docker:20.10.16-dind
   variables:
     DOCKER_TLS_CERTDIR: "/certs"
-  before_script:
-    - docker info
+    FF_NETWORK_PER_BUILD: "true"
   script:
-    - image="$(docker load --input ./builds/*docker* | cut -d' ' -f3)"
-    - docker run "$image"
+    - docker info
+    - >
+      nix-shell --run $'
+        image="$(docker load --input ./builds/*docker* | cut -d\' \' -f3)"
+        PK_TEST_DOCKER_IMAGE=$image \
+        PK_TEST_COMMAND=scripts/docker-run.sh \
+        PK_TEST_PLATFORM=docker \
+        PK_TEST_TMP_DIR=/builds/$CI_PROJECT_PATH/tmp \
```

Comment:
So the directory here has to exist under `/builds`.

Any directory that is mounted into the Polykey containers subsequently have to be under this directory.

So the issue is that normally our test code would use `os.tmpdir()` to create the data directories.

But now this has to be changed to use this location instead.

To resolve this, the tests may require an override on the data directory creation, rather than creating from `os.tmpdir()`. But don't override `TMPDIR` env variable, that should be kept as is.

Reply:
Therefore `PK_TEST_TMP_DIR` can actually be `PK_TEST_DATA_DIR`. Which would then override `os.tmpdir()`.

Reply:
`PK_TEST_TMP_DIR` is now `PK_TEST_DATA_DIR`.